### PR TITLE
fix, refactor, chore(egen): adds matplotlib library in installation step, adds code to delete locally generated files, refactors code as per new template guidelines

### PR DESCRIPTION
**Pull request by sumanvita-springml**
This PR includes the following changes:

1. Adds Colab Enterprise link and other structural changes from the new template.
2. Removes IS_TESTING from the cleaning up section.
3. Includes a dependency(matplotlib) in the installation.
4. Adds code to delete locally generated file.
5. Changes REGION to LOCATION
6. Contraction of words

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/workbench/inventory-prediction/inventory_prediction.ipynb [hunk 1]
```diff
@@ -447,7 +400,7 @@
         "\n",
         "The following cell integrates with BigQuery data from the same project through the Vertex AI's \"BigQuery in Notebooks\" integration. It can run an SQL query as it would run in the BigQuery console. \n",
         "\n",
-        "*Note:* This feature only works in a notebook running on a Vertex AI Workbench managed-notebook instance."
+        "**Note:** This feature only works in a notebook running on a Vertex AI Workbench managed-notebook instance."
```
Comment (kittyabs) [inline notebooks/official/workbench/inventory-prediction/inventory_prediction.ipynb:403]: RE: "Vertex AI Workbench managed-notebook instance", is that referencing "Vertex AI Workbench instances"?? There is no reference to "Vertex AI Workbench managed-notebook instance" in the documentation.

File: notebooks/official/workbench/inventory-prediction/inventory_prediction.ipynb [hunk 2]
```diff
@@ -799,11 +752,11 @@
         "id": "0250e32de345"
       },
       "source": [
-        "Most of the fields like discount, department, distribution center-id have a decent distribution. For the field `product_category`, there are some categories that don't constitute 2% of the dataset at least. Although there are outliers in some numerical fields, they are exempted from removing as there can be products that are expensive or belonging to a particular category that doesn't often see many sales. \n",
+        "Most of the fields like discount, department, distribution center-id have a decent distribution. For the field `product_category`, there are some categories that don't constitute 2% of the dataset at least. Although there're outliers in some numerical fields, they are exempted from removing as there can be products that are expensive or belonging to a particular category that doesn't often see many sales. \n",
```
Comment (kittyabs) [inline notebooks/official/workbench/inventory-prediction/inventory_prediction.ipynb:755]: Change back to "there are outliers" (i.e., no contraction). The "are" does have stress on it, therefore, don't contract on to "there".

**Review by kittyabs — COMMENTED** [review]
Some edits.

File: notebooks/official/workbench/inventory-prediction/inventory_prediction.ipynb (hunk 1 above)
Comment (sumanvita-springml) [reply notebooks/official/workbench/inventory-prediction/inventory_prediction.ipynb:403]: I think they are referencing "Vertex AI managed notebook instances " which will be deprecated.
 https://cloud.google.com/vertex-ai/docs/workbench/notebook-solution#managed

**Review by katiemn — APPROVED** [review]

**Pull request merged**